### ISSUE TITLE
Scroll bar and white bar at bottom of page

### ISSUE THREAD

**Issue by Cookie04DE** [fyne-io/fyne#2827]
### Describe the bug:
When opening the web version of a fyne app there is a scroll bar and when you scroll down there is a white bar. Spotted both with Safari and Chrome on macOS arm64.

### To Reproduce:
Steps to reproduce the behaviour:
1. Run `fyne serve --target=web` (Do this in a container if not on linux)
2. Open `http://localhost:8080` on both safari and chrome
3. See scroll bar and white bar at the bottom of the site

### Screenshots:
Safari:
![Safari window showing scroll bar and white bar at the bottom of the site](https://user-images.githubusercontent.com/52695768/157073782-9fe47ab3-57ff-40c9-8755-1f5a5cf5bcf1.png)
Chrome:
![Chrome window showing scroll bar and slightly smaller white bar at the bottom of the site](https://user-images.githubusercontent.com/52695768/157073775-42b06c7d-f1e1-4fd7-9646-f48dcac64c1d.png)

### Example code:
```go
package main

import (
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("Hello")
	w.SetContent(widget.NewLabel("Hello World"))
	w.ShowAndRun()
}
```

### Device (please complete the following information):
 - **OS:** MacOS (Command is run inside Linux container)
 - **Version:** Monterey 12.2.1 (21D62)
 - **Go version:** go1.17.8 linux/arm64
 - **Fyne version:** v2.1.2-rc2.0.20220306151247-e12f3a27e794

**Comment by andydotxyz** [comment]
Still lots of work to do on this driver but thanks for making us aware. Adding appropriate label

**Comment by andydotxyz** [comment]
This was fixed in 2.3 but we missed closing it
